bin.src/ci_imsim_run.py

```diff
@@ -89,8 +73,14 @@ def addArgs(cls, parser: ArgumentParser):
                             help="Whether to disable useCiLimits for deblending and process all blends")
         parser.add_argument("--config-process-singles", dest="process_singles", action="store_true",
                             help="Whether to enable processSingles (isolated objects) for deblending")
+        parser.add_argument("--config-use-skymap-small", dest="use_skymap_small", action="store_true",
```

Comment:
Why are these prefaced with `config-`?

Reply:
They have been since I committed this file (although @natelust actually wrote almost all of it). I imagine it's to indicate that they're pipeline/qgraph config options. I don't have a particularly strong opinion on the prefix other than leaning towards not changing something that's been in place this long.